manifest.go

```diff
@@ -417,6 +417,23 @@ func fromRawPruneOptions(raw rawPruneOptions) gps.RootPruneOptions {
 	return opts
 }
 
+func toRawPruneOptions(options gps.PruneOptions) rawPruneOptions {
```

Comment:
Can you change it to accept a `gps.RootPruneOptions`, so we can check if `options.ProjectOptions` is not empty?